templates/org/header.tmpl

```diff
@@ -1,6 +1,6 @@
 <div class="ui container tw-flex">
 	{{ctx.AvatarUtils.Avatar .Org 100 "org-avatar"}}
-	<div id="org-info" class="tw-flex tw-flex-col">
+	<div id="org-info" class="tw-flex tw-flex-col tw-flex-1">
```

Comment:
Maybe it also needs `tw-break-anywhere` in case the `org name` and others are too long

Reply:
Org name is limited to like 50 chars, but good to have, yes.

Reply:
Yeah works

<img width="1340" alt="image" src="https://github.com/user-attachments/assets/8924154c-d63a-4713-81f4-0acc478b97ac" />